types/transaction.go

```diff
@@ -22,13 +22,14 @@ const (
 	LegacyTx     TxType = 0x0
 	StateTx      TxType = 0x7f
 	DynamicFeeTx TxType = 0x02
+	AccessListTx TxType = 0x01
 )
```

Comment:
Order them in ascending order:
```suggestion
	LegacyTx     TxType = 0x0
	AccessListTx TxType = 0x01
	DynamicFeeTx TxType = 0x02
	StateTx      TxType = 0x7f
```